ZuluContent/Items/Misc/SpiderWeb.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using Scripts.Zulu.Utilities;
+
+namespace Server.Items
+{
+    public class SpiderWeb : Item
+    {
+        private readonly InternalTimer m_Timer;
+
+        public override bool HandlesOnMovement => true;
+
+        public override bool Decays => true;
+
+        public override string DefaultName => "Sticky Web";
+
+        public override bool OnMoveOver(Mobile mobile)
+        {
+            Stick(mobile);
+
+            return true;
+        }
+
+        public static void Stick(Mobile mobile)
+        {
+            mobile.Paralyze(TimeSpan.FromSeconds(10));
+            mobile.SendFailureMessage("You are trapped in a spider web!");
+        }
+
+        [Constructible]
+        public SpiderWeb(Point3D loc, Map map, TimeSpan duration) : base(0xEE4)
```

Comment:
Few things to fix here:

1. The `[Constructible]` attribute makes an item `[add`-able but I don't think there's a way to supply the arg types you have here from `[add`, you're better off adding a parameterless constructor here if you want to make it manually creatable.
2. Since the web is just being deleted you don't need a custom internal timer, just use `Timer.DelayCall(TimeSpan.FromMinutes(x), Delete);`. Also you should be wary about using low timer priorities (`TimerPriority.FiftyMS;`) since this will tick every 50ms to check whether a web needs to delete, which is quite noisy, once a second should be enough.
3. With effect items like this, or any transient item you need to make sure it won't survive a reboot, there's no standardise way to do this but the easiest is just a `Timer.DelayCall(TimeSpan.Zero, Delete)` in the deserialize method, or if you want them to stick around after a reboot make sure you persist the time left and restart the timer from that point on deserialize.

Reply:
Good points, was thinking that the Decayable would sort that through server reboots but this sounds better